lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
```

Comment:
naming nit: future me could easily read the name `sessionTokenVerified` and assume that it only accepts verified session tokens, then mis-use it somewhere else in the code.  Since this is a security feature I think we should be really careful about potential confusion, and a more verbose name would be better.  Maybe `"sessionTokenWithVerificationStatus"`?

Or, at the cost of a slightly bigger refactor, we could have three strategies:

* `"sessionToken"` -> looks up the token and its verification state, but doesn't check whether its verified or not
* `"sessionTokenVerified"` -> checks that the token is actually verified
* `"sessionTokenUnverified"` -> doesn't bother looking up the verification status at all

This would make it much harder for future me to accidentally do the wrong thing.

Naming is hard, but for security-related features it's doubly important that we get it right.